### ISSUE TITLE
Tales of the Abyss Lz03 Compression issue.

### ISSUE THREAD

**Issue by BWAH2leaf** [FanTranslatorsInternational/Kuriimu2#176]
Compression is working, but in game playing crashed.
( i was original file > decompress in Kuriimu2 > compress in Kuriimu2, and replace original comp file to Kuriimu2 compression file. )

You said you've modified some compression methods, could that be the reason for not being able to load?

_Originally posted by @BWAH2leaf in https://github.com/FanTranslatorsInternational/Kuriimu2/issues/173#issuecomment-902636670_

**Comment by onepiecefreak3** [comment]
No. As was already explained in the referenced issue, the files get compressed and can be decompressed into the original file without an issue, following the changes made for #173 .
You either give an explicit example (that you just replace "a" file does not help reproducing the error, you know) I can reproduce a crash or I can only judge by my tests, that showed, that the compression works again.

**Comment by onepiecefreak3** [comment]
To elaborate further what I request:
After looking into the game, I encounter many CVM files. While supported by Kuriimu2 it is important to point out that whatever your breakable file is, it is contained in such a CVM. So it's already a possibility that whatever you use to repack CVM's is causing the issue. It's also possible that whatever you use to rebuild the PS2 ISO is causing issues.
So what I want, is you telling me in which CVM, which files was replaced. With which tools you worked up until saving/recreating the PS2 ISO, and when the game actually breaks. (Could be directly on startup, or maybe somewhere ingame when said file is tried being loaded?)
